nexus/types/src/inventory/display.rs

```diff
@@ -780,6 +788,27 @@ fn display_sleds(
     Ok(())
 }
 
+fn display_ntp_status(
+    ledgered_sled_config: &OmicronSledConfig,
+    ntp_timesync: &IdOrdMap<TimeSync>,
+    f: &mut dyn fmt::Write,
+) -> fmt::Result {
+    let timesync = ledgered_sled_config
+        .zones
+        .keys()
+        .find_map(|zone_id| ntp_timesync.get(zone_id));
+
+    match timesync {
+        None => writeln!(f, "No information from NTP for this sled")?,
+        Some(ts) if ts.synced => {
+            writeln!(f, "NTP reports that time is synced")?
+        }
+        Some(_) => writeln!(f, "NTP reports that time is NOT synced")?,
+    };
```

Comment:
oh also should we report something here about how time isn't synced? maybe some kind of delta?